src/ucp/wireup/select.c

```diff
@@ -1161,7 +1165,7 @@ ucp_wireup_add_am_lane(const ucp_wireup_select_params_t *select_params,
             continue;
         }
 
-        return ucp_wireup_add_lane(select_params, am_info, UCP_LANE_TYPE_AM,
+        return ucp_wireup_add_lane(select_params, am_info, UCP_LANE_TYPE_AM, 1,
                                    select_ctx);
```

Comment:
```suggestion
        return ucp_wireup_add_lane(select_params, am_info, UCP_LANE_TYPE_AM,
                                   /*show_error*/ 1, select_ctx);
```

[minor] Don't know if this is consistent with other code or not. So feel free to ignore this comment if you don't think it's a good idea.

Reply:
ok, can add this